src/Basis/Constructors.jl

```diff
@@ -6,6 +6,12 @@
 # utility functions for generating polynomial bases
 # ------------------------------------------------------------------------------
 
+using Polynomials
+
+<<<<<<< HEAD
```

Comment:
Note: some merge/rebase conflicts are in the current commits

Reply:
Can you help me how to resolve these conflicts? Do you have some time for a call

Reply:
I have some time around 4:30 if that works, or on Monday?

Reply:
thank you very much, pls, let us do 4:30. See you, is it your same zoom link?

Reply:
Yeah, should be same link

Reply:
ok see you